### DISCUSSION TITLE
Question about gleam import

### DISCUSSION THREAD

**Discussion by rizrmd** [gleam-lang/gleam#1666] [Questions & support]
Hi! Coming from nodejs trying to learn gleam for the first time. 

In most gleam code I notice that importing a module (or a package?) is like javascript import statement. How do I know what package should I add when using an import ?

For example: 
 before using `import gleam/http` , what package should I add using `gleam add` ?

**Comment by lpil** [comment]
Hello and welcome!

The `gleam/http` module is part of `gleam_http`. https://hexdocs.pm/gleam_http/gleam/http.html

I'm afraid we don't currently have a way to find a package that contains a module, but I really like that idea :)

**Reply by rizrmd** [reply]
Maybe try like javascript did? using `from "package_name"`.

And also in vscode, I can easily cmd-click javascript import statement to go to source code. I miss that in gleam, hehe.

**Reply by inoas** [reply]
> And also in vscode, I can easily cmd-click javascript import statement to go to source code. I miss that in gleam, hehe.

That would be awesome, indeed!
